cmd/consumer/responses.go

```diff
@@ -169,7 +169,7 @@ func injectMessageResponses(ma *handler.MessageActions) {
 	)
 
 	ma.HandleStatic("block forever", "how to block forever", []string{"how to block forever"},
-		`Here's how to block forever in Go: <http://blog.sgmansfield.com/2016/06/how-to-block-forever-in-go/>`,
+		`Here's how to block forever in Go: `+"`select {}`",
```

Comment:
I've had this asked in a Go programming interview:

> Name 3 ways you can deadlock a Go program.

Some of the items in this blog post, while maybe useless trivia, satisfied that answer. I'm also 99% sure this post is why I could easily provide 3. I'm inclined to keep the post, while also including your more direct answer upfront. How would you feel about:

```
"Here's the most common way to block forever in Go: `select {}`.",
"For other ways check out this post: <https://blog.sgmansfield.com/2016/06/how-to-block-forever-in-go/>",
```

And it looks like @ScottMansfield supports HTTPS now. 😛

Reply:
That looks like a better idea